### PR TITLE
Include documentation to upgrade Gogs from source

### PR DESCRIPTION
**Pull request by joubertredrat**
This is a simple way to upgrade Gogs from source if you already have Gogs running.

### CODE REVIEW

File: en-US/upgrade/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+root: true
+name: Features
```
Comment (Unknwon) [inline en-US/upgrade/README.md]: This does not seem right...

File: TOC.ini [hunk 2]
```diff
@@ -19,6 +19,10 @@
 -: install_gogs_on_windows
 -: configuration_and_run
 
+[upgrade]
+-: README
+-: install_from_binary
```
Comment (Unknwon) [inline TOC.ini]: Not right...

**Comment by joubertredrat** [comment]
Fixed, sorry :)

**Comment by Unknwon** [comment]
Thanks!

**Comment by Unknwon** [comment]
BTW, you forgot a line here: https://github.com/joubertredrat/docs/blob/master/TOC.ini#L2

Add `-: upgrade` after `installation`

**Pull request merged**